Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/auditrepository/hibernate/AuditRepositoryDAOTest.java

```diff
@@ -85,8 +77,8 @@ public void testInsertAuditRepository() {
         Date now = new Date();
 
         AuditRepositoryRecord record = new AuditRepositoryRecord();
-        record.setMessageType("Record 1 - Message Type");
-        record.setTimeStamp(now);
+        record.setDirection("Record 1 - Message Type");
```

Comment:
I realize that we have another ticket for updating the adapter when the schema is updated.  But feel we should at least add a TODO in the unit test to make sure we don't forget to update the tests later when it's changed.